althea_kernel_interface/src/exit_server_tunnel.rs

```diff
@@ -221,7 +168,6 @@ impl dyn KernelInterface {
         local_v4: Option<(IpAddr, u8)>,
         external_v6: Option<(IpAddr, u8)>,
         exit_mesh: IpAddr,
-        external_nic: String,
```

Comment:
why are you removing external_nic?  This is currently used to recognize a gateway router.  Is the expectation that this interface be moved to the peer_interfaces instead?  How does the gateway price get used instead of the relay price?

Reply:
I moved the iptables commands from this function into the setup_nat function to keep things congruent since that function was already setting up v4 forward rules. the external_nic parameter moved with it.

Regarding using KI over libnftables json, the library wasnt able to parse some rules, for example
'nft insert rule inet fw4 input ip6 daddr . meta iifname != @rita_input add @rita_input { ip6 daddr . meta iifname counter }', if we were to use that crate we need to update the structs as well as do better error handling, I recall there being an unwrap that caused panics during testing.
In general I also think the Statement::Match syntax it uses makes it hard to read. Less external dependencies is a plus